action.yml

```diff
@@ -110,6 +114,8 @@ runs:
       run: |
         if [[ "${{ inputs.atmos-include-spacelift-admin-stacks }}" == "true" ]]; then
           atmos describe affected --file affected-stacks.json --verbose=true --repo-path "$GITHUB_WORKSPACE/main-branch" --include-spacelift-admin-stacks=true
+        elif [[ "${{ inputs.atmos-include-dependents }}" == "true" ]]; then
+          atmos describe affected --file affected-stacks.json --verbose=true --repo-path "$GITHUB_WORKSPACE/main-branch" --include-dependents=true
```

Comment:
This doesn't allow you to select both `include-spacelift-admin-stacks` and `include-dependents`, but I don't imagine we'd need both